packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.new.stories.tsx

```diff
@@ -3,8 +3,17 @@ import {facetDecorator} from '@/storybook-utils/common/facets-decorator';
 import {renderComponent} from '@/storybook-utils/common/render-component';
 import {wrapInSearchInterface} from '@/storybook-utils/search/search-interface-wrapper';
 import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit-html';
+import {within} from 'shadow-dom-testing-library';
 
-const {decorator, play} = wrapInSearchInterface();
+const {decorator, play} = wrapInSearchInterface({
+  preprocessRequest: (r) => {
+    const parsed = JSON.parse(r.body as string);
+    parsed.aq = '@filetype==("YouTubeVideo")';
```

Comment:
The idea is to ensure that the "YouTubeVideo" facet value will always be included (and in fact, be the only value) in the filetype facet (which we use as a parent).